samples/My.Hr/My.Hr.Api/My.Hr.Api.csproj

```diff
@@ -10,8 +10,13 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />

```

Comment:
`CoreEx.AutoMapper` only references `AutoMapper` - not the DI extensions, so is needed. But otherwise, you are correct that above package reference also references `AutoMapper`.